api/v1beta1/cinder_types.go

```diff
@@ -172,6 +186,32 @@ type Cinder struct {
 	Status CinderStatus `json:"status,omitempty"`
 }
 
+// DBPurge struct is used to model the parameters exposed to the Cinder cronJob
+type DBPurge struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=30
+	// +kubebuilder:validation:Minimum=1
+	// Age is the DBPurgeAge parameter and indicates the number of days of purging DB records
+	Age int `json:"age"`
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default="1 0 * * *"
+	// Schedule defines the crontab format string to schedule the DBPurge cronJob
+	Schedule string `json:"schedule"`
+}
+
+// CinderDebug contains flags related to multiple debug activities. See the
+// individual comments for what this means for each flag.
+type CinderDebug struct {
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:default=false
+	// DBSync pauses the dbSync container prior to executing the db_sync command.
```

Comment:
-1: I hate to downvote for this, but the comment is not correct, [it's an infinite sleep](https://github.com/openstack-k8s-operators/lib-common/blob/6bd6025ade373131acb7737c58cb61b43dafeee4/modules/common/const.go#L32), so it will never execute the dbsync command.
Maybe it's not the best command, and we should have a command to sleep until a file is removed, like I did in [openstack-must-gather](https://github.com/openstack-k8s-operators/openstack-must-gather/blob/main/collection-scripts/gather_debug), but it's what we have right now.

Reply:
Changing "prior to executing" to "instead of executing" per our conversation.